index-flow.yml

```
@@ -0,0 +1,57 @@
+jtype: Flow
+version: '1'
+with:
+  protocol: 'http'
+  cors: True
+  port_expose: $JINA_PORT
+executors:
+  - name: frame_extractor
+    uses: jinahub://VideoLoader/v0.1
+  - name: image_filter
+    uses: FilterModality
+    uses_with:
+      modality: 'image'
+    py_modules:
+      - executors.py
+    needs: frame_extractor
+  - name: image_encoder
+    uses: jinahub+docker://AudioCLIPImageEncoder/v0.3
+    uses_with:
+      traversal_paths: ['c',]
+    volumes: $MODEL_MOUNT_CACHE
+    needs: image_filter
+  - name: image_indexer
+    uses: jinahub://SimpleIndexer/v0.4
+    uses_metas:
+      workspace: $JINA_WORKSPACE/image_indexer
+    volumes: $WORKSPACE_MOUNT
+    needs: ['image_encoder']
+  - name: audio_filter
+    uses: FilterModality
+    uses_with:
+      modality: 'audio'
+    py_modules:
+      - executors.py
+    needs: frame_extractor
+  - name: audio_segmenter
+    uses: AudioSegmenter
+    uses_with:
+      traversal_paths: ['c',]
```

reduce the overlaps between the chunks.
```suggestion
      traversal_paths: ['c',]
      chunk_strip: 10
```